### DISCUSSION TITLE
Running activesetup seems to not working With SCCM on current logged on user

### DISCUSSION THREAD

**Discussion by FRacine** [PSAppDeployToolkit/PSAppDeployToolkit#998] [General]
Hi,

Anybody notice the latest 3.10.1 PSADTK, is not applying activesetup script when ran with sccm on the current logged on user? If the script is run manually then everything is looking fine. I suspect something with the system account. Or it is the french os language?!

Thanks,

**Comment by DanGough** [comment]
This should be resolved in 3.10.2!
